saleor/graphql/discount/mutations/promotion/promotion_update.py

```diff
@@ -142,3 +145,22 @@ def send_promotion_toggle_webhook(
             cls.call_event(event, instance)
             instance.last_notification_scheduled_at = now
             instance.save(update_fields=["last_notification_scheduled_at"])
+            if event == manager.promotion_started:
+                return "started"
+            if event == manager.promotion_ended:
+                return "ended"
+        return None
+
+    @classmethod
+    def save_events(
```

Comment:
The method name is misleading. It suggests making only save, but you also optionally trigger webhook